practice-app/app/location.py

```diff
@@ -0,0 +1,72 @@
+from .views import app, session, Base
+from flask import render_template, request
+from flask_login import login_required, current_user
+from sqlalchemy import Column, String, ForeignKey, Float, Integer
+from sqlalchemy.orm import Mapped, mapped_column
+import os
+import googlemaps
+
+
+class Location(Base):
+    __tablename__ = 'LocationTable'
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    address = Column(String(50))
+    latitude = Column(Float)
+    longitude = Column(Float)
+    user_id: Mapped[int] = mapped_column(ForeignKey('User.id'), nullable=True)
+
+
+try: 
+    GOOGLEMAPS_API_KEY = os.environ["GOOGLEMAPS_API_KEY"]
```

Comment:
Can you add this environment variable into .env.example file

Reply:
I added